sniffles/ruletrafficgenerator.py

```diff
@@ -985,6 +985,58 @@ def updateSequence(self, dir="to server", data_len=1):
                 self.current_seq_b_to_a += data_len
                 self.current_ack_a_to_b = self.current_seq_b_to_a
 
+# !!Create background traffic.
+class BackgroundTraffic(TrafficStream):
+    def __init__(self, rule=None, sconf=None, start_sec=-1, start_usec=0):
+        super().__init__()
+        self.proto = rule.getProto()
+        self.sport = Port(rule.getSport())
+        self.dport = rule.getDport()
+        self.rule = rule
+        self.rand = False
+        self.current_seq_a_to_b = random.randint(0, 4000000000)
+        self.current_ack_a_to_b = 0
+        self.current_seq_b_to_a = random.randint(0, 4000000000)
+        self.current_ack_b_to_a = 0
+
+        if sconf:
```

Comment:
If you pass in the sconf, then I don't think that need to duplicate these values.